manifests/params.pp

```diff
@@ -48,7 +48,23 @@
 
   case $::osfamily {
     'Debian': {
-      $service_provider = 'upstart'
+
+      if $install_method == 'archive' {
+        $manage_unzip = true
+      }
+
+      case $::lsbdistcodename {
+        /(jessie|stretch|sid|xenial|yakketi|zesty)/: {
```

Comment:
I have not tested them and did not include them for that reason, however I'm assuming they work, as there should not be much difference between Debian and Ubuntu.